packages/pds/src/db/index.ts

```diff
@@ -193,8 +193,17 @@ export class Database {
     return txRes
   }
 
-  async txAdvisoryLock(name: string): Promise<boolean> {
+  async takeTxAdvisoryLock(name: string): Promise<boolean> {
     this.assertTransaction()
+    return this.txAdvisorylock(name)
+  }
+
+  async checkTxAdvisoryLock(name: string): Promise<boolean> {
+    this.assertNotTransaction()
+    return this.txAdvisorylock(name)
+  }
+
+  private async txAdvisorylock(name: string): Promise<boolean> {
```

Comment:
Just a tiny consistency thing.
```suggestion
    return this.txAdvisoryLock(name)
  }

  async checkTxAdvisoryLock(name: string): Promise<boolean> {
    this.assertNotTransaction()
    return this.txAdvisoryLock(name)
  }

  private async txAdvisoryLock(name: string): Promise<boolean> {
```

Reply:
yup good catch